ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -122,6 +122,17 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.neighbors[neighbor].route_map_in is arista.avd.defined %}
    neighbor {{ neighbor }} route-map {{ router_bgp.neighbors[neighbor].route_map_in }} in
 {%         endif %}
+{%         if router_bgp.neighbors[neighbor].default_originate is arista.avd.defined %}
+{%             set neighbor_default_originate_router_bgp = "neighbor " ~ peer_group ~ " default-originate" %}
+
```

Comment:
Please remove this empty line. This will affect the template output.

Reply:
nice catch, I will fix this and update the molecule asap.